drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -22,7 +22,7 @@ def self.options_for_type(pick_list_type, user:, project_id: nil)
       when 'COC'
         selected_project = Hmis::Hud::Project.viewable_by(user).find_by(id: project_id) if project_id.present?
         available_codes = if selected_project.present?
-          selected_project.project_cocs.pluck(:CoCCode)
+          selected_project.project_cocs.pluck(:CoCCode).uniq.map { |code| [code, ::HUD.cocs[code] || code] }
```

Comment:
Pretty sure you're going to want to include the CoC Number in the display, something like:
`Tucson/Pima County CoC (AZ-501)`

Is that handled on the front-end, or is it worth adding here?

Reply:
That happens on line 31 below. The label looks like `AZ-501 - Tucson/Pima County CoC`